src/Xamarin.Android.Build.Tasks/Xamarin.Android.FSharp.targets

```diff
@@ -42,6 +42,12 @@ Copyright (C) 2012 Xamarin. All rights reserved.
     <Import
       Condition="'$(Language)' != 'F#' And Exists('$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\FSharp\Microsoft.FSharp.Targets')"
       Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\FSharp\Microsoft.FSharp.Targets" />
+    <Import
+      Condition="'$(Language)' != 'F#' And Exists('$(FSharpInstallDir)Microsoft.FSharp.Targets')"
+      Project="$(FSharpInstallDir)Microsoft.FSharp.Targets" />
+    <Import
```

Comment:
Is the order important for these imports? Which should be preferred? I assume if both exist the last one will be used (as it will overwrite all the bits from the other.. maybe)

Reply:
@dellis1972 I think `$(Language)` gets set to `F#` as soon as one is imported, so it should pick the first one that loads.

Reply:
> Is the order important for these imports?

Yes.

As @jonathanpeppers notes, the `'$(Language)' != 'F#'` check ensures that the first "existing" `<Import/>` "wins." As soon as a file is found an imported, the expectation is that file will set `<Language>F#</Language>`, which will prevent subsequent imports from actually importing.

Reply:
That said, I have no idea what the order *should* be here. :-(

Reply:
@jonpryor @dellis1972 

I tested what it's doing here a little bit.

If you are inside VS, it loads this one because `$(VisualStudioVersion)` is set: `$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\FSharp\Microsoft.FSharp.Targets`

If you are MSBuild command line, it picks the newest F# you have installed, such as: `$(MSBuildExtensionsPath32)\..\Microsoft SDKs\F#\4.1\Framework\v4.0\Microsoft.FSharp.Targets`

My `FSHARPINSTALLDIR` is set to `C:\Program Files (x86)\Microsoft SDKs\F#\4.1\Framework\v4.0\`.

So perhaps the order should be:
* Try the VS one
* Try the environment variable
* Try 4.1
* Try 4.0
* Try 3.1

I happened to have code in this PR like in this order already. Thinking it through a little more, what do you guys think?